instant/src/test/java/com/adyen/checkout/instant/DefaultInstantPaymentDelegateTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 9/11/2022.
+ */
+
+package com.adyen.checkout.instant
+
+import app.cash.turbine.test
+import com.adyen.checkout.components.model.paymentmethods.PaymentMethod
+import com.adyen.checkout.components.repository.PaymentObserverRepository
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.core.log.Logger
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.util.Locale
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class DefaultInstantPaymentDelegateTest {
+
+    private lateinit var delegate: DefaultInstantPaymentDelegate
+
+    @BeforeEach
+    fun before() {
+        val configuration = InstantPaymentConfiguration.Builder(
+            Locale.US,
+            Environment.TEST,
+            TEST_CLIENT_KEY
+        ).build()
+        delegate = DefaultInstantPaymentDelegate(PaymentObserverRepository(), PaymentMethod(type = TYPE), configuration)
+        Logger.setLogcatLevel(Logger.NONE)
+    }
+
+    @Test
+    fun `when subscribed then component state flow should propagate a valid state`() = runTest {
+        delegate.componentStateFlow.test {
+            with(awaitItem()) {
+                Assertions.assertEquals(TYPE, data.paymentMethod?.type)
```

Comment:
Minor: you can statically import `assertEquals` and `assertTrue` since we do it elsewhere and it looks a tiny bit nicer